src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -240,11 +247,11 @@ public synchronized static void parse(DLNAMediaInfo media, InputFile inputFile,
 						}
 
 						value = MI.Get(audio, i, "BitDepth");
-						if (!value.isEmpty()) {
-							try {
+						if (isNotBlank(value) || !value.contains("Bit")) {
+							if (!value.contains("/")) {
 								currentAudioTrack.setBitsperSample(Integer.parseInt(value));
-							} catch (NumberFormatException nfe) {
```

Comment:
I agree, but i don't really how `try...catch` work with `if..else` so i limited the fix to my actual skill. That's why.